config/manifests/bases/mongodb-kubernetes.clusterserviceversion.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     capabilities: Deep Insights
     categories: Database
     certified: "true"
-    containerImage: quay.io/mongodb/mongodb-kubernetes:1.0.1
+    containerImage: quay.io/mongodb/mongodb-kubernetes:1.1.0
```

Comment:
I expected another change here:

```
replaces: mongodb-kubernetes.v1.0.1
```

We have `scripts/evergreen/release/update_helm_values_files.py` (part of -pre-commit) should've updated it. Could you please double check why it didn't happen?

Reply:
I've updated the `replaces:` value in this PR for now.

Reply:
@m1kola plz take a look